packages/ui-manager/tsconfig.json

```diff
@@ -14,11 +14,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
Are you sure you need this alias ? Cause you don't use @/translations in your component